recipes/opensim/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+cd opensim-core-source
+cd ..
+mkdir build_dep
+cd build_dep
+cmake ../opensim-core-source/dependencies -LAH \
+    -DCMAKE_OSX_SYSROOT=${CONDA_BUILD_SYSROOT} \
+    -DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET} \
+    -DCMAKE_BUILD_TYPE="Release" \
+    -DSUPERBUILD_ezc3d=ON \
```

Comment:
I assume superbuild is vendored this package. THere is a feedstock we should use for this, if possible: https://github.com/conda-forge/ezc3d-feedstock. If not, there can't be any clashes if someone also has the conda forge ezc3d installed.

Reply:
If it only works with one version of ezc3d, then you can vendor it, but is it install only in the opensim source tree so there would be no clobbering if exc3d is also installed sepearately?

You can pin the version in the requires section, but that will also create incompatibilities.

Ideally CMake is setup so that you can specify the path of all the dependencies, so that we use the conda forge binaries.

Reply:
From the first comment box: " Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)." That's the ideal situation. So having `ezc3d==1.4.6` in the requirements is the first thing to try. But yes, opensim's build script would need to support pointing to an arbitrary path for the package.